scc/ootb-supply-chain-testing-scanning.hbs.md

```diff
@@ -325,6 +325,34 @@ Create workload:
      15 + |        branch: main
      16 + |      url: https://github.com/sample-accelerators/tanzu-java-web-app
 ```
+## <a id="cve-triage-workflow"></a> CVE Triage Workflow
```

Comment:
Whichever page serves as the primary source for this content, it would be good to cross-link from the scan policy, troubleshooting, and supply chain pages to aid discoverability.

Reply:
Added links to the cve triage workflow in the supply chain page in the scanpolicy and troubleshooting pages